clients/mobile/src/setup/notification/saveDeviceToken.ts

```diff
@@ -1,10 +1,18 @@
 import { Platform } from 'react-native'
+import authService from '../../authentication/services/authService'
+import StorageService from '../../utility/storageService'
+import { TOKEN } from '../constant/token'
 
 export const saveDeviceTokenToSNS = async(
```

Comment:
the name of this wrong

saveDeviceTokenOnSNS

Reply:
I didn't touch the existing function name. will change.

Reply:
resolved in [9a0e8d2a6](https://github.com/CraftsmenLtd/BloodConnect/pull/272/commits/9a0e8d2a659996bbf0a9183fdf1252e7f074ca9f)